app/code/Bulbulatory/Recomendations/Controller/Adminhtml/Recommendations/Delete.php

```diff
@@ -1,32 +1,33 @@
 <?php
 namespace Bulbulatory\Recomendations\Controller\Adminhtml\Recommendations;
 
-use \Magento\Framework\App\Action\Context;
+use Magento\Backend\App\Action\Context;
 use Magento\Framework\Controller\ResultFactory;
-use Bulbulatory\Recomendations\Model\ResourceModel\Recommendation\CollectionFactory;
+use Bulbulatory\Recomendations\Api\RecommendationRepositoryInterface;
 
 class Delete extends \Magento\Backend\App\Action
 {
+    private $_recommendationRepositoryInterface;
```

Comment:
Do zmiennej nie będzie przypisany interfejs, tylko już instancja konkretnej klasy która interfejs implementuje, więc lepiej nazwać ją po prostu `$_recommendationRepository`
```suggestion
    private $_recommendationRepository;
```